README.rst

```diff
@@ -68,5 +65,18 @@ Prerequisites
 
 If you want to run traitsui, you must also install:
 
-* `traits <https://github.com/enthought/traits>`_
+* `traits <https://github.com/enthought/traits>`__
```

Comment:
Because then it is not a "named link" in restructured text.  This was causing PyPI to not render the text correctly, because we had two "traits" named links in the README.

Yes, it's obscure.  Took some Googling to work out what was going wrong.  Try running `python setup.py check --restructuredtext -s` with and without to see the difference.